quinn/src/tests.rs

```diff
@@ -1,13 +1,15 @@
-use super::{read_to_end, ClientConfigBuilder, Endpoint, NewStream, ServerConfigBuilder};
-use futures::{Future, Stream};
-use slog::{Drain, Logger, KV};
 use std::{
```

Comment:
While you're at it, maybe normalize this to the single-line uses we've got elsewhere, too?